spec/git_reflow/2-lgtm/git_reflow_spec.rb

```diff
@@ -0,0 +1,668 @@
+require 'spec_helper'
+
+describe GitReflow do
+  let(:git_server)       { GitReflow::GitServer::GitHub.new {} }
+  let(:github)           { Github.new basic_auth: "#{user}:#{password}" }
+  let(:user)             { 'reenhanced' }
+  let(:password)         { 'shazam' }
+  let(:oauth_token_hash) { Hashie::Mash.new({ token: 'a1b2c3d4e5f6g7h8i9j0', note: 'hostname.local git-reflow'}) }
+  let(:repo)             { 'repo' }
+  let(:base_branch)      { 'master' }
+  let(:feature_branch)   { 'new-feature' }
+  let(:enterprise_site)  { 'https://github.reenhanced.com' }
+  let(:enterprise_api)   { 'https://github.reenhanced.com' }
+  let(:hostname)         { 'hostname.local' }
+
+  let(:github_authorizations)  { Github::Client::Authorizations.new }
+  let(:existing_pull_requests) { Fixture.new('pull_requests/pull_requests.json').to_json_hashie }
+  let(:existing_pull_request)  { GitReflow::GitServer::GitHub::PullRequest.new existing_pull_requests.first }
+
+  before do
+
+    # Stubbing out numlgtm value to test 2 LGTM reviewers in gitconfig file
+    GitReflow::Config.stub(:get).with("constants.numlgtm").and_return(2)
+    GitReflow::Config.stub(:get).and_call_original
+
```

Comment:
you may want to use `stub_command_line`([see here](https://github.com/simonzhu24/gitreflow/blob/master/spec/support/command_line_helpers.rb#L2)) to avoid any accidental local command calls :wink: